scripts/build.sh

```diff
@@ -19,4 +19,5 @@ cp -f ./websites/index.html ./public
 
 echo "Running link checker"
 
-yarn check-links
+# run the test but don't fail the otherwise successful build (separate check exists) -> || true
+yarn check-links || true
```

Comment:
🤔 what's the point of having it here if we have a check anyway?

Reply:
laziness to think about it  :-(.  Can be removed